### DISCUSSION TITLE
Foot Sliding

### DISCUSSION THREAD

**Discussion by Amorill-21** [Sixze/ALS-Refactored#187] [Q&A]
How does foot locking work in ALS refactored?

I'm trying to get a different walk forward animation for when my character is in combat, the animation play correctly but when moving the feet start gliding for a few frames and aren't planted on the ground. I tried to recreate the logic of the BS_Als_WalkRun_Forward blendspace but haven't had any success. Does anyone know how the foot locking works and how I can get it to work with a different animation?

**Comment by Alex-G** [comment]
It uses Control Rig afaik.
